package.json

```diff
@@ -36,14 +36,14 @@
   "dependencies": {
     "merge": "1.2.x",
     "stylelint-config-prettier": "^4.0.0",
-    "stylelint-order": "~0.8.1",
+    "stylelint-order": "1.0.0",
     "stylelint-prettier": "^0.2.2",
-    "stylelint-scss": "^3.0.0"
+    "stylelint-scss": "3.3.0"
   },
   "devDependencies": {
-    "eslint": "^4.19.1",
-    "eslint-plugin-shopify": "^20.0.0",
-    "stylelint": "^9.1.1",
+    "eslint": "^5.4.0",
+    "eslint-plugin-shopify": "24.0.0-alpha.1",
```

Comment:
temporarily using the alpha here, this will be updated once https://github.com/Shopify/eslint-plugin-shopify/pull/151 is merged and shipped.

Reply:
👍 Living on the edge